lxd/migration/migrate.pb.go

```diff
@@ -2,8 +2,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.1
-// 	protoc        v3.21.12
+// 	protoc-gen-go v1.28.0
+// 	protoc        v3.20.1
```

Comment:
What's the reason for this change? I don't think we should downgrade the protobuf version.

Reply:
Mmm yeah looks like my protoc  version is older. Another reason to dislike that protobuf dependency.

Reply:
I'll see if I can get a newer one on my system somehow

Reply:
@stgraber @monstermunchkin that is fixed now.